src/web-file/file.js

```diff
@@ -0,0 +1,92 @@
+
+export default class WebFile {
+    constructor (options = {}) {
+        this._fileName = options.fileName;
+        this._maxSize = options.maxSize;
+        this._messagesQueue = [];
+        this._isRunning = false;
+        this._fsLink = null;
+    }
+
+    push (string) {
+        this._messagesQueue.push(string);
+        if (!this._isRunning) {
+            this._isRunning = true;
+            this._writeAttempt();
+        }
+    }
+
+    _writeAttempt () {
+        this._requestFile((windowFsLink, fileEntry, fileWriter) => {
+            fileWriter.onwriteend = this._onWriteEnd.bind(this);
+            if (fileWriter.length > this._maxSize) {
+                this._rotateLogs(windowFsLink, fileEntry, fileWriter);
+            } else {
+                this._appendQueueData(fileWriter);
+                this._messagesQueue = [];
+            }
+        });
+    }
+
+    _onWriteEnd () {
+        if (this._messagesQueue.length) {
+            this._writeAttempt();
+        } else {
+            this._isRunning = false;
+        }
+    }
+
+    _requestFile (callback) {
+        this._requestFileSystem(fsLink => this._requestFileWriter(fsLink, callback));
+    }
+
+    _requestFileSystem (callback) {
+        if (this._fsLink) {
+            callback(this._fsLink);
+            return;
+        }
+        try {
+            var requestFs = window.webkitRequestFileSystem || window.requestFileSystem;
```

Comment:
..so it will take `webkitRequestFileSystem`.
Unprefixed globals are preferred over prefixed